examples/kernel_ridge/plot_krc_custom_kernel.py

```diff
@@ -0,0 +1,61 @@
+"""
+===============================================
+Kernel Ridge Classifier with custom kernel
+===============================================
+
+Simple usage of KRC to classify a sample. It will
+plot the decision surface and the support vectors.
+
+"""
+print(__doc__)
+
+import numpy as np
+import matplotlib.pyplot as plt
+from sklearn import datasets
+from sklearn.kernel_ridge import KernelRidgeClassifier
+
+# Import some data to play with
+iris = datasets.load_iris()
+# We only take the first two features. We could
+# avoid this ugly slicing by using a two-dim dataset
+X = iris.data[:, :2]
+Y = iris.target
+
+
+def my_kernel(X, Y=None):
+    """
+    We create a custom kernel:
+
+                 (2  0)
+    k(X, Y) = X  (    ) Y.T
+                 (0  1)
+    """
+    if Y is None:
+        Y = X
+    M = np.array([[2, 0], [0, 1.0]])
+    return np.dot(np.dot(X, M), Y.T)
+
+
+h = .02  # step size in the mesh
+
+# we create an instance of Kernel Ridge Classifier and fit out data.
+clf = KernelRidgeClassifier(kernel=my_kernel)
+clf.fit(X, Y)
+
+# Plot the decision boundary. For that, we will assign a color to each
+# point in the mesh [x_min, x_max]x[y_min, y_max].
+x_min, x_max = X[:, 0].min() - 1, X[:, 0].max() + 1
+y_min, y_max = X[:, 1].min() - 1, X[:, 1].max() + 1
+xx, yy = np.meshgrid(np.arange(x_min, x_max, h), np.arange(y_min, y_max, h))
+Z = clf.predict(np.c_[xx.ravel(), yy.ravel()])
+
+# Put the result into a color plot
+Z = Z.reshape(xx.shape)
+plt.pcolormesh(xx, yy, Z, cmap=plt.cm.Paired)
+
+# Plot also the training points
+plt.scatter(X[:, 0], X[:, 1], c=Y, cmap=plt.cm.Paired, edgecolors='k')
+plt.title('3-Class classification using Support Vector Machine with custom'
```

Comment:
More generally I feel like this might be an unnecessary proliferation of examples, which may make it harder for users to find the information they need.

Reply:
I see... so, linking with with [main review comment](https://github.com/scikit-learn/scikit-learn/pull/10633#pullrequestreview-100589899), you suggest to unify SVM and Ridge plot and examples, right?

Personally, I don't think that is a good idea, since every test and example is organized by folders, so many folders would be removed and/or merged. Besides, unifying SVR/SVC (which calls to `libsvm`in C) and Kernel Ridge/Classifier could be caotic since the implementation in scikit-learn is different.

Reply:
On one hand, **SVM** narrative and minimization **problem is geometrical**, which does not allow multiclassification. Besides, SVM thinks about *how to build an hyperplane that separates class 1 from class 2*, and different tricks are applied in order to make it multiclass and also available for regressions.

On the other hand, **Ridge** narrative and minimization **problem is analytical**, because it tries to look for the *w* that 

![image](https://user-images.githubusercontent.com/17336458/36843667-ec285332-1d4f-11e8-9c7e-2a9b3bed6fce.png)

using Tikhonov regularization directly in the minimization problem. 

So they either does not start from the same hypothesis or achieve to the same solution. That's why I've introduced Kernel Ridge Classifier in the documentation expanding Kernel Ridge ( https://github.com/scikit-learn/scikit-learn/pull/10633/files#diff-242cc346ae920fe5058a877fbae888e8 )

Reply:
Basically I don't know that we need more examples. What we need is to make clear in the narrative documentation (`doc/` rather than `examples/`) that we have alternative kernel-based classifiers available. That documentation, and/or an example, should help a user decide (a) whether a kernel classifier is a good choice for them; (b) whether SVC or KRC is a better choice for them. Beyond that, how does an extended example help?

Reply:
Ok, 00b85b6be0bce0850ac19907d1f01a580ec5f0dc and 23a7e37 (new commit) are aimed to provide documentation as you requested

Reply:
So what of the redundant example?